ci-operator/step-registry/upi/install/powervs/cluster/upi-install-powervs-cluster-commands.sh

```diff
@@ -194,32 +194,35 @@ function cleanup_prior() {
         sleep 60
         echo "Done Deleting the ${CRN}"
     done
-    echo "Delete network ocp-net on powerVS region"
+
     # Dev: functions don't work inline with xargs
-    HOME=${IBMCLOUD_HOME_FOLDER} ic pi subnet ls | grep -v ocp-net | awk '{print $1}' | xargs -I {} ibmcloud pi subnet delete {} --force || true
+    echo "Delete network non-'ocp-net' on PowerVS region"
+    export IBMCLOUD_HOME=${IBMCLOUD_HOME_FOLDER}
+    ibmcloud pi subnet ls | grep -v ocp-net | awk '{print $1}' | xargs -I {} ibmcloud pi subnet delete {} --force || true
+    echo "Done deleting non-'ocp-net' on PowerVS"
 
     # VPC Instances
     # VPC LBs 
-    # TODO: FIXME - need to be selective so as not to blow out other workflows being run
+    WORKSPACE_NAME="multi-arch-comp-${LEASED_RESOURCE}-1"
```

Comment:
we have already declared in configure_terraform L263 and L264 for Workspace and VPC names . Still do we need again here .

Reply:
Apart from the above , everything looks good .